### PR TITLE
Update 1st arguments

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: flatpak.plugin.zsh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-alias fti="flatpak install"
+#alias fti="flatpak install"
```
Comment (bilelmoussaoui) [inline flatpak.plugin.zsh]: any reason for this change? if you want to remove it, please do so in a separate commit

Comment (Ferenc-) [reply flatpak.plugin.zsh]: Right. That wasn't meant to be part of the commit. Removed.
